### PR TITLE
documentation v2

### PR DESCRIPTION
**Pull request by ignacioct**
Referencing #26 

What is already done (and we can review it):

- [x]  Simplify URL and folder structure by removing documentation from the path
- [x] Home page index.rst should be what's now Introduction to Rubrix. Ideally, this page will still be the first item of the Getting started block (I've seen this in other docs) but if it's not easy to do, no problem.
- [x] Rename Introduction to Rubrix -> Rubrix docs
- [x] Merge Concepts and Methods into one section called Concepts (I think is a heading issue).
- [x] Complete missing things in Concepts (keep it as simple as possible).

To do:

- [x] Add right-side table of contents (toc) to each page, with a header like "On this page".
- [x] Try to include the video on the home page with autoplay and using either html5 or a sphinx-contrib package (I don't know if the video is already made)
- [x] Complete the home page (what's now Intro to Rubrix)
- [x] Add Community section, with direct links to github and github discussion forum.

### CODE REVIEW

File: docs/getting_started/concepts.rst [hunk 1]
```diff
@@ -2,92 +2,134 @@
...
-   ner_record = rb.TextClassificationRecord(
-       annotation=[],
-     prediction=[]
+   record = rb.TextClassificationRecord(
+        inputs={
+            "text": "your input text"
+        },
+        multi_label = True,
+
+        prediction = [('label1', 0.8), ('label2', 0.2)],
```
Comment (dvsrepo) [inline docs/getting_started/concepts.rst]: Change scores to reflect this is is multi label distribution with at least three labels summing more than 1.0.

In both examples use more real input.text and labels. Example: SPAM and HAM input.text "Free discounts bla bla" reading the example we should get an idea of a real model prediction.

File: docs/getting_started/concepts.rst [hunk 2]
```diff
@@ -2,92 +2,134 @@
...
 
 Prediction
-----------
+^^^^^^^^^^
 
 A prediction is a piece information assigned to a record, a label or a set of labels and typically by a machine process.
 
 Metadata
---------
+^^^^^^^^
```
Comment (dvsrepo) [inline docs/getting_started/concepts.rst:127]: This part is missing, please complete

**Review by dvsrepo — CHANGES_REQUESTED** [review]

File: docs/getting_started/concepts.rst (hunk 1 above)
Comment (ignacioct) [reply docs/getting_started/concepts.rst]: Should I also reference real annotators and prediction agents? I could use some HuggingFace models for prediction agents and some fake names for annotation agents.

**Comment by ignacioct** [comment]
To do: 

- [ ] Decide if we want real-world annotation agent and prediction agent
- [x] Right-side toc as a local table of content (posposed, waiting for support #30 )
- [x] Include youtube-uploaded video to index.

**Comment by ignacioct** [comment]
@dvsrepo I've the changes I made to the concepts example are ok, I think we can merge. Tell me what you think

**Review by dvsrepo — APPROVED** [review]
💯 thanks @ignacioct . I think we can merge this

**Pull request merged**